src/custom/pages/App/AppMod.tsx

```diff
@@ -31,10 +31,13 @@ import DarkModeQueryParamReader from 'theme'
 // import Vote from './Vote'
 // import VotePage from './Vote/VotePage'
 
+import Footer from 'components/Footer'
+
 const AppWrapper = styled.div`
   display: flex;
   flex-flow: column;
   align-items: flex-start;
+  min-height: 100vh;
```

Comment:
this matches the `height` of the body wrapper to the vh of page and keeps our footer down without `position: fixed`-ing it. Position fixed is great until the content is in the way of any large height element, i.e Markdown, and makes everything feel bad man.